packages/core/src/controllers/OptionsController.ts

```diff
@@ -68,7 +69,11 @@ export const OptionsController = {
   },
 
   setEnableAnalytics(enableAnalytics: OptionsControllerState['enableAnalytics']) {
```

Comment:
Since `setEnableAnalytics` is called very early on in web3modal lifecycle, wouldn't this always resolve to else clause while api is fetching the flag?

Reply:
Good catch! Last commit fixes this 🙏